integration/tools/validation/src/main/java/alluxio/fuse/correctness/client/RandomReadClient.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.correctness.client;
+
+import alluxio.fuse.correctness.Constants;
+import alluxio.fuse.correctness.Utils;
+
+import com.google.common.base.Preconditions;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+/**
+ * This class is a client verifying the correctness of sequential read of AlluxioFuse.
+ */
+public class RandomReadClient implements Runnable {
+  private final String mLocalFilePath;
+  private final String mFuseFilePath;
+  private final int mBufferSize;
+
+  /**
+   * Creates an instance of {@link RandomReadClient}.
+   *
+   * @param localFilePath
+   * @param fuseFilePath
+   * @param bufferSize
+   */
+  public RandomReadClient(String localFilePath, String fuseFilePath, int bufferSize) {
+    mLocalFilePath = Preconditions.checkNotNull(localFilePath);
+    mFuseFilePath = Preconditions.checkNotNull(fuseFilePath);
+    mBufferSize = bufferSize;
+  }
+
+  @Override
+  public void run() {
+    try (RandomAccessFile localRandomFile = new RandomAccessFile(mLocalFilePath, "r");
```

Comment:
For each random read validation, e.g. num threads = 64
for each file size + each buffer size + each read type, create 64 threads to execute the validation.
In the each of the 64 thread validation, use the while loop to execute until timeout, each execution creates one thread and creates two related randomAccessFile/FileInStream/FileOutStream resources to execute the single random read operation or sequential read operation. Does it sound reasonable?

Reply:
I see. Previously I didn't realize using 64 threads is to stress the AlluxioFuse, which is why I didn't care about the resource creation/destroy